backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/search/predicate/impl/ElasticsearchMatchIdPredicateBuilder.java

```diff
@@ -43,13 +45,18 @@
 	private static final JsonObjectAccessor IDS = JsonAccessor.root().property( "ids" ).asObject();
 	private static final JsonAccessor<JsonElement> VALUES = JsonAccessor.root().property( "values" );
 	private List<String> values = new ArrayList<>();
+	private final ToIndexIdValueConverter idConverter;
```

Comment:
This type should not be raw. If we need to make it raw in order to compile, then maybe the generic parameter should not be there.